extensions/amp-story-interactive/0.1/amp-story-interactive-quiz.js

```diff
@@ -94,8 +94,12 @@ export class AmpStoryInteractiveQuiz extends AmpStoryInteractive {
     // Localize the answer choice options
     const localizationService = Services.localizationForDoc(this.element);
     this.answerChoiceOptions_ = this.answerChoiceOptions_.map((choice) => {
-      return localizationService.getLocalizedString(
-        LocalizedStringId[`AMP_STORY_QUIZ_ANSWER_CHOICE_${choice}`]
+      return (
```

Comment:
nit: can we use `||` instead of `|` for consistency with other parts of the codebase?

Reply:
Done